qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -367,9 +369,72 @@ def get_spinZ_matrix_elements(mol_name, hf_data, n_active_electrons=None, n_acti
     return spinz_matrix_elements
 
 
+def particle_number(n_orbitals, docc_orb=None, act_orb=None, mapping="jordan_wigner"):
```

Comment:
Notice that I simplified the signature of the function. There is no internal dependence on the OpenFermion class `MolecularData`. Interestingly, this simplification allows the function to build the particle-number operator for any quantum system, not only molecules. This is an important thing to keep in mind to make PL-QChem to stand out, as suggested by Nathan.